.prettierignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
Can't put this in package.json :( https://github.com/prettier/prettier/issues/3460

Decided it was easier to add a file than create a more complicated glob in `format` npm script